canvas_modules/harness/cypress/support/properties/properties-verification-cmds.js

```diff
@@ -223,3 +224,195 @@ Cypress.Commands.add("verifyTipForValidationIconInSummaryPanel", (summaryPanelId
+		.eq(1)
+		.invoke("text")
+		.then((text) => expect(text.trim()).to.equal(value));
+});
+
+Cypress.Commands.add("verifySummaryPanelValueForParameterInConsole", (testValue, parameterName) => {
+	// Verify that the event log has a value of "is_date" for the "expressionCellTable" parameter
+	cy.document().then((doc) => {
+		const lastEventLog = testUtils.getLastEventLogData(doc);
+		const parameterValues = lastEventLog.data.form[parameterName];
+		let found = false;
+		for (var idx = 0; idx < parameterValues.length; idx++) {
+			const parameter = parameterValues[idx];
+			for (var indx = 0; indx < parameter.length; indx++) {
+				if (parameter[indx] === testValue) {
+					found = true;
+					break;
+				}
+			}
+		}
+		expect(found).to.equal(true);
+	});
+});
+
+Cypress.Commands.add("verifyIconInSubPanel", (iconName, panelName) => {
+	cy.getWideFlyoutPanel(panelName)
+		.then((wideFlyoutPanel) => {
+			cy.wrap(wideFlyoutPanel)
+				.find(".properties-expression-validate")
+				.then((validateButton) => {
+					if (iconName === "none") {
+						cy.wrap(validateButton)
+							.find(".validateIcon")
+							.should("not.exist");
+					} else {
+						cy.wrap(validateButton)
+							.find(".validateIcon")
+							.then((icon) => {
+								cy.wrap(icon)
+									.find("svg")
+									.invoke("attr", "class")
+									.then((iconClass) => {
+										expect(iconClass).to.include(iconName);
+									});
+							});
+					}
+				});
+		});
+});
+
+Cypress.Commands.add("verifyRowInSelectColumnsTable", (tableId, fieldName, rowNumber, panelName) => {
+	cy.getWideFlyoutPanel(panelName)
+		.find(`div[data-id='properties-ft-${tableId}']`)
+		.find("div[role='properties-data-row']")
+		.eq(rowNumber - 1)
+		.find(".properties-readonly")
+		.should("have.text", fieldName);
+});
+
+Cypress.Commands.add("verifyFieldIsSelectedInFieldPickerPanel", (fieldName, dataType, panelName) => {
+	// Following logic works based on assumption  - fieldName in each row is unique
+	let rowNumber;
+	cy.getWideFlyoutPanel(panelName)
+		.find("div[role='properties-data-row']")
+		.each(($el, index) => {
+			if ($el[0].childNodes[1].textContent === fieldName) {
+				rowNumber = index;
+				return false;
+			}
+		})
+		.then((rows) => {
+			cy.wrap(rows)
+				.eq(rowNumber)
+				.then((row) => {
+					// Verify field name
+					cy.wrap(row)
+						.find(".properties-fp-field-name")
+						.should("have.text", fieldName);
+					// Verify dataType
+					cy.wrap(row)
+						.find(".properties-fp-field-type")
+						.should("have.text", dataType);
+					// Verify checkbox is selected
+					cy.wrap(row)
+						.find(".properties-vt-row-checkbox")
+						.find("input")
+						.should("be.checked");
+				});
+		});
+});
+
+Cypress.Commands.add("verifyFieldsInStructureListEditorTable", (tableId, fields, rowNumber, columnNumber) => {
+	cy.get(`div[data-id='properties-ft-${tableId}']`)
+		.find(`div[data-id='properties-structurelist_sub_panel_${rowNumber}_${columnNumber}']`)
```

Comment:
`structurelist_sub_panel` is part of the propertyId.  This shouldn't be hard coded.  The method name can also be more generic since this isn't structurelisteditor specific.